src/submission/submission-to-clinical/stat-calculator.ts

```diff
@@ -183,18 +191,23 @@ const getEmptyCoreStats = (): CoreCompletionStats => {
   });
 };
 
-const isCoreEntitySchemaName = (clinicalType: string) =>
+const isCoreEntitySchemaName = (clinicalType: string): clinicalType is CoreClinicalSchemaName =>
   coreClinialSchemaNamesSet.has(clinicalType as CoreClinicalSchemaName);
 
 function noNeedToCalcCoreStat(
   donor: Donor,
   clinicalType: CoreClinicalSchemaName,
   forceFlags: ForceRecaculateFlags,
 ) {
+  // if recalculate ovveriden, need to ignore completion value since overriden value could be 100%
```

Comment:
This case was not handled earlier which caused a bug where overridden is not removed after submission